buildingmotif/dataclasses/model.py

```diff
@@ -157,6 +164,10 @@ def validate(
             ontologies are missing (i.e. they need to be loaded into BuildingMOTIF), defaults
             to True
         :type error_on_missing_imports: bool, optional
+        :param engine: the engine to use for validation. "pyshacl" or "topquadrant". Using topquadrant
+            requires Java to be installed on this machine, and the "topquadrant" feature on BuildingMOTIF,
+            defaults to "pyshacl"
+        :type engine: str, optional
```

Comment:
Again, I'm wondering if this configuration should be moved to a single place. Not having to specify which engine you want to use every time seems good? especially if validate is called from another function within the SDK you'd have to make sure you passed the engine through every layer of the function call.

Reply:
Done this as well!